build.cake

```diff
@@ -26,108 +15,76 @@ using Newtonsoft.Json.Linq;
 
 var TARGET = Argument ("t", Argument ("target", "Default"));
 var BUILD_CONFIG = Argument ("config", "Release");
-var MAX_CPU_COUNT = Int32.Parse(Argument("maxcpucount", "0"));
+var MAX_CPU_COUNT = Argument("maxcpucount", 0);
```

Comment:
Auto-conversion based on the type of the default value - integer `0` instead of string `"0"`